app/Domain/Dice/Dice.cpp

```diff
@@ -0,0 +1,53 @@
+//
+// Created by Gabriel Noriega on 2018-02-14.
+//
+
+#include <random>
+#include "Dice.h"
+#include <iomanip>
+
+using std::cout;
+using std::endl;
+
+
+int Dice::generateRandomNumber() {
+    const int range_from = 0;
```

Comment:
Suggested to pull range [0,6] out of the `generateRandomNumber` function, as private members of the `Dice` class, or as parameters to the function `roll()` that are passed along to `generateRandomNumber()`

Reply:
You're right. I'll modify them as private members.